realsense2_camera/include/base_realsense_node.h

```diff
@@ -59,8 +59,8 @@ using realsense2_camera_msgs::msg::Extrinsics;
 using realsense2_camera_msgs::msg::IMUInfo;
 
 #define FRAME_ID(sip) (static_cast<std::ostringstream&&>(std::ostringstream() << _camera_name << "_" << STREAM_NAME(sip) << "_frame")).str()
-#define IMU_FRAME_ID (static_cast<std::ostringstream&&>(std::ostringstream() << _camera_name << "_imu_frame")).str()
-#define IMU_OPTICAL_FRAME_ID (static_cast<std::ostringstream&&>(std::ostringstream() << _camera_name << "_imu_optical_frame")).str()
+#define IMU_FRAME_ID(sip) (static_cast<std::ostringstream&&>(std::ostringstream() << _camera_name << "_imu_frame")).str()
+#define IMU_OPTICAL_FRAME_ID(sip) (static_cast<std::ostringstream&&>(std::ostringstream() << _camera_name << "_imu_optical_frame")).str()
```

Comment:
I don't see usage of the new parameter..

Reply:
From my test this parameter allow usage of the updated values of the variable "_camera_name". As you can see in #2617, before this modification, the default value of "_camera_name" was used. After this modification, the value of "_camera_name" used was the one defined in the parameters. I did this modification based on the others lines around that are using a similar pattern.

Reply:
What I meant is that your change added a macro parameter that is not used.
If we look on other macro's i.e.
```
#define FRAME_ID(sip) (static_cast<std::ostringstream&&>(std::ostringstream() << _camera_name << "_" << STREAM_NAME(sip) << "_frame")).str()
```
You can notice the macro usage of the parameter here:
```
STREAM_NAME(sip) 
```

Reply:
You are right this does not seem to be doing the difference. Thanks you for following with me.